Documentation/reference/extensions/DangerDSL.md

```diff
@@ -5,137 +5,61 @@
 ## Properties
 ### `fails`
 
-```swift
-public var fails: [Violation]
-```
-
```

Comment:
hrm, I must have missed something in that command in the makefile?

Reply:
The `Makefile` looks good to me. Could be a bug or a change in a lower dependency. I’d be happy to take a look. 

Any idea of which version was used to generate the documentation before?

Reply:
It's locked to 0.5.1 - `.package(url: "https://github.com/eneko/SourceDocs.git", from: "0.5.1"),`